src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -1,3 +1,5 @@
+{% set colors = { 'success': '#4F805D', 'warning': '#A46A1F', 'error': '#B0413E' } %}
```

Comment:
I suggest adding a comment saying this should stay in sync with the variable defined in ``profiler.css.twig`` (and having a similar comment there, so that both sides are warning people). I will avoid mistakes where only one of them is updated